1.5.0 Release

alx-alexpark opened the pull request:
New mods:
- Canary
- Eureka Ships
- Kotlin for Forge (Dependency)
- RFTools {Base,builder,dimensions,storage,utility}
- Valkyrien-Skies
- XNet
- Cooking for blockheads
- Applied cooking
- Applied-energistics-2-wireless-terminals
- Attribute fix
- Clumps
- Everycompat
- Just Enough Advancements
- Just Enough Effect Descriptions
- Roughly Enough Resources
- Tax Free Levels

alx-alexpark committed:
add new mods to prepare for 1.5.0

456dev committed:
revert updating mods that use create 0.5.1
Create steam and rails will not update until the create fabric update is out
https://github.com/Layers-of-Railways/Railway/issues/116
https://github.com/DragonsPlusMinecraft/CreateEnchantmentIndustry/issues/75
Create goggles doesn't have an issue, looks like its dropped 1.18 support

456dev committed:
update mods
ae2 got updated twice, with different file hashes and the same name :/

updated:
- ae2
- applied-energistics-2
- bookshelf
- controlling
- create
- createaddition
- integrated-dynamics
- jade-addons
- roughly-enough-items
- supermartijn642s-core-lib

456dev committed:
Add 13 mods, change overloaded armor bar to use forge version

456dev committed:
upgrade forge version
causes crashes, as some mods have already updated past it

456dev committed:
add selene (used by everycomp), update ferritecore config to prevent crash with canary

the456gamer commented:
my additions for this are still being tested, its on branch the456gamer-1.5.0-mods
ill pr to this branch after, at which point this can be merged

456dev committed:
make ferritecore config not overwrite others

456dev committed:
update quark

456dev committed:
remove aeinfinitybooster

456dev committed:
Change to locally compiled FarmersDelight-1.18.2-1.2.1.jar
includes https://github.com/unilock/FarmersDelight/commit/55436e92ac641c5de5c1e34f94a17fc7c5870467

source here: https://github.com/the456gamer/FarmersDelight

456dev committed:
add default config for applied cooking lib, removing onJoin message

456dev committed:
move custom farmersdelight to build from source github action

alx-alexpark committed:
Merge pull request #20 from hackclub/the456gamer-1.5.0-mods

1.5.0 changes

alx-alexpark commented:
This closes #15 #11 #12 when merged.

456dev committed:
Add simple backups mod
